src/components/views/elements/Field.js

```diff
@@ -46,6 +46,9 @@ export default class Field extends React.PureComponent {
         // and a `feedback` react component field to provide feedback
         // to the user.
         onValidate: PropTypes.func,
+        // If specified, contents will appear as a tooltip on the element and
+        // validation feedback tooltips will be suppressed.
+        tooltip: PropTypes.node,
```

Comment:
Maybe `tooltipContent`, as it's not literally the tooltip itself? Or `tooltipLabel`?